charts/vela-core/templates/defwithtemplate/webservice.yaml

```diff
@@ -329,8 +329,8 @@ spec:
         	}]
 
         	// +ignore
-        	// +usage=Specify what kind of Service you want. options: "ClusterIP", "NodePort", "LoadBalancer", "ExternalName"
-        	exposeType: *"ClusterIP" | "NodePort" | "LoadBalancer" | "ExternalName"
```

Comment:
Why delete the ExternalName here?

Reply:
This option can not work, miss a required parameter.